rhel/usr_lib_systemd_system_ovsdb-server-ic.service

```diff
@@ -0,0 +1,32 @@
+# See ovn-ic(8) for details about ovn-ic.
+#
+# To customize the ovsdb-server-ic service, you may create a configuration file
+# in the /etc/systemd/system/ovsdb-server-ic.d/ directory.  For example, to specify
+# additional options to be passed to the "ovn-ctl start_ic_ovsdb" command, you
+# could place the following contents in
+# /etc/systemd/system/ovsdb-server-ic.d/local.conf:
+#
+#   [System]
+#   Environment="OVSDB_SERVER_IC=--db-ic-nb-create-insecure-remote=yes --db-ic-sb-create-insecure-remote=yes"
+#
+# Alternatively, you may specify environment variables in the file /etc/sysconfig/ovsdb-server-ic:
+#
+#   OVSDB_SERVER_IC=--db-ic-nb-create-insecure-remote=yes --db-ic-sb-create-insecure-remote=yes"
+
+[Unit]
+Description=OVN global (IC) northbound and southbound OVSDB servers
+After=syslog.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+Environment=OVN_RUNDIR=%t/ovn OVN_DBDIR=/var/lib/ovn
+EnvironmentFile=-/etc/sysconfig/ovn
+EnvironmentFile=-/etc/sysconfig/ovsdb-server-ic
+ExecStartPre=-/usr/bin/chown -R ${OVN_USER_ID} ${OVN_DBDIR}
+ExecStart=/usr/share/ovn/scripts/ovn-ctl \
+          --ovn-user=${OVN_USER_ID} start_ic_ovsdb $OVSDB_SERVER_IC
```

Comment:
@numansiddique, should I rename OVSDB_SERVER_IC to OVSDB_SERVER_IC_OPTS?

Reply:
Sounds good to me.